tests/test_ec2/test_key_pairs.py

```diff
@@ -29,6 +29,7 @@
 
 RSA_PUBLIC_KEY_RFC4716 = b"""\
 ---- BEGIN SSH2 PUBLIC KEY ----
+cOmmENt: moto@github.com
```

Comment:
Can we have a few more tests with example RFC4716-compliant keys here, to verify that they all work?
I'm thinking with comment, without any comment, with commend + `x-comment`, etc.

I noticed that the spec has a few examples: 
https://www.rfc-editor.org/rfc/rfc4716#section-3.6